data/autoyast_sle15/pc_tools.xml

```diff
@@ -0,0 +1,159 @@
+                <name>Public Cloud Devel</name>
+                <ask_user config:type="boolean">false</ask_user>
+                <selected config:type="boolean">true</selected>
+            </listentry>
+            <listentry>
+                <media_url>https://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_15/</media_url>
+                <product>python_devel</product>
+                <alias>Python Devel</alias>
+                <product_dir>/</product_dir>
+                <priority config:type="integer">20</priority>
+                <ask_on_error config:type="boolean">false</ask_on_error>
+                <confirm_license config:type="boolean">false</confirm_license>
+                <name>Python Devel</name>
+                <ask_user config:type="boolean">false</ask_user>
+                <selected config:type="boolean">true</selected>
+            </listentry>
+        </add_on_products>
+    </add-on>
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email/>
+        <reg_code>{{SCC_REGCODE}}</reg_code>
+        <install_updates config:type="boolean">true</install_updates>
+        <addons config:type="list">
+            <addon>
+                <name>sle-module-basesystem</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+            <addon>
+                <name>sle-module-public-cloud</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+            <addon>
+                <name>sle-module-development-tools</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+            <addon>
+                <name>sle-module-legacy</name>
+                <version>15</version>
+                <arch>x86_64</arch>
+            </addon>
+        </addons>
+    </suse_register>
+    <bootloader>
+        <device_map config:type="list">
+            <device_map_entry>
+                <firmware>hd0</firmware>
+                <linux>/dev/vda</linux>
+            </device_map_entry>
+        </device_map>
+        <global>
+            <timeout config:type="integer">1</timeout>
+            <append>splash=verbose</append>
+        </global>
+    </bootloader>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+    <software>
+        <products config:type="list">
+            <product>SLES</product>
+        </products>
+         <packages config:type="list">
+            <package>netcat-openbsd</package>
+            <package>python3-img-proof</package>
+            <package>python3-img-proof-tests</package>
+            <package>git-core</package>
+            <package>ntp</package>
+            <package>gcc</package>
+            <package>python3-pip</package>
+            <package>aws-cli</package>
```

Comment:
Valid point. Still ideal it would be to avoid the need of prepare_tools.pm. But I guess this isn't possible.
Let's say we are going to install all tools on `prepare_tools.pm` then also move `python3-img-proof*`, or?

Reply:
I agree. If we go keep packages here, I'd keep common things here and install all the public cloud related packages in prepare_tools.

Reply:
Ok, I will add this in my PR.

Reply:
Done in #8023.

Reply:
> Valid point. Still ideal it would be to avoid the need of prepare_tools.pm. But I guess this isn't possible.
everything is possible it is just matter of enough time and good will. So the only question do we want to fully drop prepare_tools or not ? my answer "Yes I do". and yours ? @jlausuch , @cfconrad , @ldevulder